SpatialGDK/Source/Private/Interop/GlobalStateManager.cpp

```diff
@@ -39,6 +83,27 @@ void UGlobalStateManager::ApplyUpdate(const Worker_ComponentUpdate& Update)
 	}
 }
 
+void UGlobalStateManager::ApplyMapUpdate(const Worker_ComponentUpdate& Update)
+{
+	Schema_Object* ComponentObject = Schema_GetComponentUpdateFields(Update.schema_type);
+
+	if (Schema_GetObjectCount(ComponentObject, 1) == 1)
+	{
+		SetDeploymentMapURL(GetStringFromSchema(ComponentObject, 1));
+	}
+
+	if (Schema_GetBoolCount(ComponentObject, SpatialConstants::GLOBAL_STATE_MANAGER_ACCEPTING_PLAYERS_ID) == 1)
+	{
+		bool bUpdateAcceptingPlayers = bool(Schema_GetBool(ComponentObject, SpatialConstants::GLOBAL_STATE_MANAGER_ACCEPTING_PLAYERS_ID));
```

Comment:
static_cast<>

Reply:
`!!`

Reply:
Asked above, I prefer this c-style cast. What's the advantage of the other two?